tests/playwright-test/ui-mode-trace.spec.ts

```diff
@@ -583,6 +583,36 @@ test('fails', async ({ page }) => {
     `.trim());
 });
 
+// https://github.com/microsoft/playwright/issues/35999
+test('should update git diff between test runs', async ({ runUITest, writeFiles, initGitRepo }) => {
```

Comment:
make it a proper annotation, like here: https://github.com/microsoft/playwright/blob/9b9c300c297aa27c26c9fb36471a72e5cfe7506a/tests/playwright-test/ui-mode-trace.spec.ts#L154